tests/pipeline/test_buyback_auth.py

```diff
@@ -34,16 +34,16 @@
     BlazeShareBuybackAuthorizationsLoader,
 )
 from zipline.pipeline.loaders.utils import (
-    get_values_for_date_ranges,
     zip_with_floats,
     zip_with_dates
 )
 from zipline.testing.fixtures import (
     WithPipelineEventDataLoader, ZiplineTestCase
 )
 
-date_intervals = [[None, '2014-01-04'], ['2014-01-05', '2014-01-09'],
-                  ['2014-01-10', None]]
+date_intervals = [
```

Comment:
Because of the refactoring. There happens to only be a single sid with non-NaN data in this test, but in other tests, the list has 3 levels, and I'm using the same method on all of them.